content/news/XXXX-XX-XX-mixxx-2.4.2-released.md

```diff
@@ -0,0 +1,143 @@
+title: Mixxx 2.4.2 Release Announcement
+authors: Evelynne Veys
+tags: update, 2.4.2
+comments: yes
+status: draft
+
+#### Winter and Summer are coming
+
+Contrasts.
+As the leaves fall and the first snowflakes start drifting down in the North, while the southern skies start warming up
+with summer sun, we are thrilled to finally release Mixxx 2.4.2! Finally because a bit later than planned, perfection

```

Comment:
tripped me up during reading because I thought it was part of the sentence instead of basically quoting the earlier one.
```suggestion
with summer sun, we are thrilled to finally release Mixxx 2.4.2! "Finally" because a bit later than planned, perfection
```

Reply:
if it makes it easier to read...